test/e2e/common/runtimeclass.go

```diff
@@ -117,16 +117,15 @@ func newRuntimeClassPod(runtimeClassName string) *v1.Pod {
 }
 
 func expectPodRejection(f *framework.Framework, pod *v1.Pod) {
+	_, err := f.ClientSet.NodeV1beta1().RuntimeClasses().Get(*pod.Spec.RuntimeClassName, metav1.GetOptions{})
+	gomega.Expect(apierrs.IsNotFound(err)).To(gomega.BeTrue(), "should not be exist")
 	// The Node E2E doesn't run the RuntimeClass admission controller, so we expect the rejection to
 	// happen by the Kubelet.
```

Comment:
does the node e2e tests run the RuntimeClass controller?

Reply:
yes, apiserver enabled RuntimeClass controller